package.json

```diff
@@ -10,13 +10,16 @@
   "funding": "https://github.com/kitajs/html?sponsor=1",
   "license": "Apache-2.0",
   "author": "arthurfiorette <npm@arthur.place>",
-  "sideEffects": ["register.js"],
+  "sideEffects": [
+    "register.js"
+  ],
   "main": "index.js",
   "types": "index.d.ts",
   "scripts": {
     "bench": "tsc -p tsconfig.benchmark.json && cp benchmark/package.json dist/benchmark/package.json && cd dist/benchmark && node index.js",
+    "exec-test": "c8 --reporter lcov --reporter text node --trace-warnings --enable-source-maps --test dist/test",
     "format": "prettier --write .",
-    "test": "tsc && c8 --reporter lcov --reporter text node --trace-warnings --enable-source-maps --test dist/test",
+    "test": "(tsc && pnpm exec-test) && (tsc -p ./tsconfig.jsxruntime.json && pnpm exec-test)",
```

Comment:
This tests code, but not ours. When double transpiling with react-jsx, the only thing being tested is if `tsc` is traspiling correctly, which I'm sure microsoft/typescript has better tests than these.

Instead, you could create a test suite to ensure the file `/jsx-runtime.js` is well typed and exports `jsx`, `jsxs` and `Fragment`.

Reply:
please also remove tsconfig.jsxruntime.json and exec-test script

Reply:
See https://github.com/kitajs/html/pull/129#discussion_r1495005802